introduction.md

```diff
@@ -1,2 +1,3 @@
-out of Ruby, JavaScript, HTML & CSS, iOS or Electives I would take HTML
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

Reply:
simply don't understand what you want me to do